### PR TITLE
GhostStead org and source

### PR DESCRIPTION
**Pull request by ghoststead**
Add an organization and source for GhostStead (https://www.ghoststead.com/)

### CODE REVIEW

**Comment by ghoststead** [comment]
@googlebot I signed it!

File: organizations/ghoststead.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "id": "EBAY",
```
Comment (pfrisella) [inline organizations/ghoststead.json:2]: Add the correct the organization details.

**Review by pfrisella — CHANGES_REQUESTED** [review]

File: organizations/ghoststead.json (hunk 1 above)
Comment (ghoststead) [reply organizations/ghoststead.json]: Done.  Force pushed for fix the previous broken commit.

**Review by pfrisella — APPROVED** [review]
LGTM. Thanks for adding!

**Pull request merged**